app/views/categories/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<header>
+  <nav class="nav navbar-light bg-primary">
+    <div class="container py-1 px-3 d-flex justify-content-between align-items-center">
+    <div></div>
+    <h1 class="fs-6 fw-normal text-white pt-2 pg-title">CATEGORIES</h1>
+    <small><%= link_to 'Log Out', destroy_user_session_path, method: :delete, class: "text-white text-decoration-none bg-transparent border-0 fw-normal" %></small>
+    </div>
+  </nav>
+</header>
+
+<section class="container-fluid space">
+    <% if @categories.empty? %>
+       <p class="w-75 d-flex justify-content-center align-items-center mx-auto text-center no-category">
+        You do not have any categories yet<br> Kindly create one with the "Add A Category" button below
+      </p>
+
+      <% else %>
+        <ul class="m-0 p-0 mb-5 list-unstyled">
+          <% @categories.each do |category| %>
+            <%= render partial: 'category_list', locals: { category: category } %>
+          <% end %>
+        </ul>
+    <% end %>
+</section>
+
+<footer class="bg-success p-2 mt-5 fixed-bottom ">
+  <div class="w-100 d-flex justify-content-center align-items-center bg-transparent border border-white">
+  <%= link_to 'Add A Category', new_category_path, class: ' text-white p-2 text-decoration-none stretched-link' %>
+  </div>
+</footer>
```

Comment:
- [ ] Also consider changing the color of the button like the one in the design guideline to follow it 

| Your design  | Design guidline |
| ------------- | ------------- |
|![image](https://user-images.githubusercontent.com/53402600/227622164-9ca2da9b-7f1f-4bf4-ae40-f3f31684071c.png)  | ![image](https://user-images.githubusercontent.com/53402600/227622204-a9c3cec5-9018-4073-97e5-7e6a12ad76bf.png) |